app/scripts/components/boilerplate/header.jsx

```diff
@@ -10,8 +10,9 @@ const Header = React.createClass({
     return (
       <div className="header">
         <div className="logo">
-          <h1><T k="site-name" /></h1>
+          <img src="images/tz-flag.png"/>
```

Comment:
also @gkalvatcheva make paths to static components aboslute, so like `/images/...`.

That way, we can, maybe, later, set up the app routing to use the real URL path (so the app might be loaded from somewhere like `/waterpoints/some-waterpoint-id`), and links like this will still work.